### PR TITLE
Faker

### PR DESCRIPTION
**Pull request by rthai**
Hey @lhmun, requesting your code review for my faker data generator.
Includes:
data generator
data generator test
json file creation
some minor changes and additions to dependencies and ignore files

Thanks!

### CODE REVIEW

File: db/writeJSONFile.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          stream.write(']');
+          stream.end();
+        });
+      } else {
+        ok = stream.write(generateData(), 'utf-8');
+        stream.write(',');
+      }
+    } while (i > 0 && ok);
+    if (i > 0) {
+      // had to stop early!
```
Comment (louismun) [inline db/writeJSONFile.js]: Consider getting rid of these comments

File: db/generateFakerData.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const getRandomNum = function () {
+  const min = 1;
+  const max = 6;
+  return Math.floor(Math.random() * ((max - min) + 1)) + min;
+};
+
+const generateReviews = function () {
+  const reviews = [];
+  for (let i = 0; i < getRandomNum(); i += 1) {
```
Comment (louismun) [inline db/generateFakerData.js]: You can use faker number generator here instead of creating an entire function.

**Review by louismun — CHANGES_REQUESTED** [review]
Looks good, just a couple changes.

File: db/generateFakerData.js (hunk 2 above)
Comment (rthai) [reply db/generateFakerData.js]: Of course! Changed.

File: db/writeJSONFile.js (hunk 1 above)
Comment (rthai) [reply db/writeJSONFile.js]: Removed.

**Review by louismun — APPROVED** [review]

**Comment by rthai** [comment]
Thanks Louis!

**Pull request merged**